prov/efa/src/efa_mr.c

```diff
@@ -103,19 +106,27 @@ static int efa_mr_cache_reg(struct fid *fid, const void *buf, size_t len,
 			    struct fid_mr **mr_fid, void *context)
 {
 	struct efa_domain *domain;
-	struct efa_mr *md;
+	struct efa_mr *efa_mr;
 	struct ofi_mr_entry *entry;
+	uint64_t core_access;
 	int ret;
 
 	struct iovec iov = {
 		.iov_base	= (void *)buf,
 		.iov_len	= len,
 	};
 
+	/* To support Emulated RMA path, if the access is not supported
+	 * by EFA, modify it to FI_SEND | FI_RECV
+	 */
+	core_access = access;
+	if (!access || (access & ~EFA_MR_SUPPORTED_PERMISSIONS))
```

Comment:
I don't think we need to change access to core_access here. Because we do that conversion inside efa_mr_alloc().

Reply:
+1